Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -11,7 +11,12 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
 
     override var profile: Profile! {
         didSet {
-            profile.bookmarks.modelForRoot(self.onNewModel, failure: self.onModelFailure)
+            // Until we have something useful to show for desktop bookmarks,
+            // only show mobile bookmarks.
+            // Note that we also need to build a similar kind of virtual hierarchy
+            // to what we have on Android.
+            profile.bookmarks.modelForFolder(BookmarkRoots.MobileFolderGUID, success: self.onNewModel, failure: self.onModelFailure)
```

Comment:
I think I'd like to move all of the guid/folders to an enum, but i'm fine with lots of small follow ups for non user facing stuff.